### ISSUE TITLE
`\`NgxSkeletonLoaderModule\` is already loaded and it might cause issues. To avoid that, import the module only once in your app.

### ISSUE THREAD

**Issue by shadowdestiny** [willmendesneto/ngx-skeleton-loader#85]
In the latest version 3.0, an error appears with the following comment: "` \ `NgxSkeletonLoaderModule \` is already loaded and it might cause issues. To avoid that, import the module only once in your app. ".

I am currently using Angular 9 and NPM. I have modularized my web application under the following structure:
-app.module.ts
-profile.module.ts 
-shared.module.ts 
in the shared.module.ts is the call to the library "NgxSkeletonLoaderModule", everything goes well until entering the other module "profile.module.ts "the reported error is generated, displaying the described message. The strangest thing is that only the shared.module.ts is the only module whose function is to share the components for the rest of the modules and it is the only module where the call is made to the NgxSkeletonLoaderModule.

It works perfectly when I comment out the line in the file:  node_modules\ngx-skeleton-loader\fesm2015\ngx-skeleton-loader.js 

`/*if (parentModule) {
            throw new Error(`\`NgxSkeletonLoaderModule\` is already loaded and it might cause issues. To avoid that, import the module only once in your app.`);
        }*/`

Node v14.15.4
Angular 9
ngx-skeleton-loader ^3.0.0
chrome

**Comment by feracios120499** [comment]
i have exactly the same problem. Rolled back to version 2.10.1 and everything works

**Comment by shadowdestiny** [comment]
> 2.10.1
Thanks for commenting, I tried exactly as you comment and it worked for me with version 2.10.1. Thanks for the input :)

**Comment by willmendesneto** [comment]
Hi @shadowdestiny  and @feracios120499 . Thanks for raising this issue.

Actually, this was added since it was raised in another issue and at that time was logical. However, I can see this shouldn't be the case.

Based on that, I reverted the code changes and published a new version for the module. If for some reason other consumers need to have this validation approach, they might need to wrap `NgxSkeletonLoaderModule` into a new module and add the parent load check in this wrap module.

Feel free to upgrade your project to use `ngx-skeleton-loader@4.0.0` and get the fix/rollback in place.

Once again, thanks for that!
